duckduckgo/privacy-dashboard
# Malware status support and design updates - macOS

Asana: https://app.asana.com/0/1206594217596623/1208788075506205/f

**Reviewer:** @shakyShane 



## Description:

- Support for Malware protection on all platforms
- Visual updates to Phishing & Malware view

## Steps to test this PR:

1. Visit the [malware](https://deploy-preview-250--ddgdash.netlify.app/app-debug/html/iframe?platforms=macos%2Cwindows&state=malware) or [phishing](https://deploy-preview-250--ddgdash.netlify.app/app-debug/html/iframe?platforms=macos%2Cwindows&state=phishing) previews
2. Copy, layout and functionality should match [Figma](https://www.figma.com/design/uP27mEGEaHCI7ZYXAs1815/Safe-Browsing-Warnings?node-id=1347-68241&m=dev)
